src/attributes.cpp

```diff
@@ -2196,7 +2197,7 @@ namespace attributes {
     }
 
     std::string CppPackageIncludeGenerator::getHeaderGuard() const {
-        return "__" + packageCpp() + "_h__";
+        return "RCPP_" + packageCpp() + "_h_gen_";
```

Comment:
To confirm, does this mean:

`"_rcpp_" + packageCpp() + "_h_gen_`

or

`"_RCPP_" + packageCpp() + "_h_gen_`?